### PR TITLE
Zoom to fit

### PR DESCRIPTION
**Pull request by fsih**
### Resolves
Fixes https://github.com/LLK/scratch-paint/issues/216
Demo at https://llk.github.io/scratch-gui/zoomToFit/

### Proposed Changes
- Zoom to fit when first opening a sprite in the costume editor
- Zoom level is saved per-sprite
zoomtofit

### Reason for Changes
Editing to-scale sprite is difficult, hopefully this will make it easier

### CODE REVIEW

File: src/reducers/zoom-levels.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        });
+    default:
+        return state;
+    }
+};
+
+// Action creators ==================================
+const saveZoomLevel = function (zoomLevel) {
+    if (!(zoomLevel instanceof paper.Matrix)) {
+        log.warn(`Not a matrix: ${zoomLevel}`);
```
Comment (chrisgarrity) [inline src/reducers/zoom-levels.js:31]: Should this continue to save a zoom level if it's not a matrix?

**Comment by fsih** [comment]
I made the rename and documentation changes we discussed.

**Review by chrisgarrity — DISMISSED** [review]
Looks good!

File: src/reducers/zoom-levels.js (hunk 1 above)
Comment (fsih) [reply src/reducers/zoom-levels.js:31]: Hmm, I'm not sure how to fail gracefully here. I don't think I can not return an action

**Comment by fsih** [comment]
I'm going to hold off on merging because I think the idea is that we'll try this on its own branch during the smoke test, and then decide whether to merge it afterwards.

**Pull request merged**